packages/core/src/App/Components/Layout/Footer/complaints-policy.jsx

```diff
@@ -0,0 +1,243 @@
+                    components={[<span className='complaints-policy__list-item-prefix' key={0} />]}
+                />,
+            ],
+        },
+        {
+            title: localize('4. Awards and orders'),
+            list: [
+                <Localize
+                    key={0}
+                    i18n_default_text='<0>a.</0>The decisions made by the DRC are binding on us. DRC decisions are binding on you only if you accept them.'
+                    components={[<span className='complaints-policy__list-item-prefix' key={0} />]}
+                />,
+                <Localize
+                    key={1}
+                    i18n_default_text='<0>b.</0>If you agree with a DRC decision, you will need to accept it within 14 days. If you do not respond to the DRC decision within 14 days, the complaint is considered closed.'
+                    components={[<span className='complaints-policy__list-item-prefix' key={0} />]}
+                />,
+                <Localize
+                    key={2}
+                    i18n_default_text='<0>c.</0>We must award the settlement within 28 days of when the decision is reached.'
+                    components={[<span className='complaints-policy__list-item-prefix' key={0} />]}
+                />,
+                <Localize
+                    key={3}
+                    i18n_default_text='<0>d.</0>If the decision is made in our favour, you must provide a release for us within 7 days of when the decision is made, and the complaint will be considered closed.'
+                    components={[<span className='complaints-policy__list-item-prefix' key={0} />]}
+                />,
+            ],
+        },
+        {
+            title: localize('5. Disclaimer'),
+            content: {
+                text: localize(
+                    'The Financial Commission accepts appeals for 45 days following the date of the incident and only after the trader has tried to resolve the issue with the company directly.'
+                ),
+            },
+        },
+    ];
+
+    const modal_content = [...policy_content, ...complaints_procedure].map((row, index) => (
+        <div key={index} className='complaints-policy__section'>
+            <div className='complaints-policy__section-title'>{row.title}</div>
+            {row.list && (
+                <div className='complaints-policy__list'>
+                    {row.list.map((item, i) => (
+                        <div key={i} className='complaints-policy__list-item'>
+                            {item}
+                        </div>
+                    ))}
+                </div>
+            )}
+            {row.content && (
+                <div className='complaints-policy__section-content'>
+                    {row.content.text && row.content.text}
+                    {row.content.subcontent &&
+                        row.content.subcontent.map((item, i) => (
```

Comment:
Is it possible to use `optional chaining` here and remove the check on line 205?
```suggestion
                        row.content.subcontent?.map((item, i) => (
```